srcalc/src/test/java/gov/va/med/srcalc/domain/model/SampleModels.java

```diff
@@ -291,6 +291,33 @@ public static DiscreteNumericalVariable wbcVariable()
         return var;
     }
     
+    public static DiscreteNumericalVariable cardiacAgeVariable()
+    {
+    	final Category ageLessThan50 = new Category(
```

Comment:
This code indentation is all over the place. I think the time has come for you to fix your Eclipse. Did you import that formatter config?